src/app/metadata/services/metadataStats.service.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Short module description
+ * @copyright The GHGA Authors
+ * @license Apache-2.0
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { computed, inject, Injectable, resource, Signal } from '@angular/core';
+import { ConfigService } from '@app/shared/services/config.service';
+import { firstValueFrom, map } from 'rxjs';
+import {
+  BaseGlobalSummary,
+  emptyGlobalSummary,
+  GlobalSummary,
+} from '../models/global-summary';
+
+/**
+ * Metadata Stats service
+ *
+ * This service provides the functionality to fetch the global metadata summary stats from the server.
+ */
+@Injectable({
+  providedIn: 'root',
+})
```

Comment:
This service is now only used in the GlobalSummaryComponent so I think it makes to sense remove the providedIn: root and instead add it to the providers array of that component.